charts/testkube-api/executors.json

```diff
@@ -3,7 +3,7 @@
     "name": "tracetest-executor",
     "executor": {
       "executorType": "job",
-      "image": "kubeshop/testkube-tracetest-executor:f62d60b",
+      "image": "kubeshop/testkube-tracetest-executor:{{ default .Chart.AppVersion .Values.executor.image.tag }}",
```

Comment:
Maybe we should just use the `image.tag` here as these are always built together and this just complicate our sandbox config.